cos_alerter/alerter.py

```diff
@@ -128,6 +128,12 @@ def initialize():
                 "notify_time": None,
             }
 
+    @staticmethod
+    def clients():
+        """Return a list of clientids."""
```

Comment:
[nit] should the function not be called 'clientids' or 'client_ids' then?

Reply:
We use "clients" throughout the code already so I'm just going to leave this. Perhaps going through and changing everything at some point would be a good idea.